File: c++/0014-longest-common-prefix/main.cpp
#include <cstdlib>
#include <iostream>
#include <vector>

#include "solution.cpp"

using namespace std;

int main() {
    Solution solution;

    vector<string> strs1 = {"flower", "flow", "flight"};
    cout << "fl: " << solution.longestCommonPrefix(strs1) << endl;

    vector<string> strs2 = {"dog", "racecar", "car"};
    cout << "Nothing: " << solution.longestCommonPrefix(strs2) << endl;

    return EXIT_SUCCESS;
}

File: java/0014-longest-common-prefix/Main.java
public class Main {
    public static void main(String[] args) {
        Solution solution = new Solution();

        System.out.println("fl: " + solution.longestCommonPrefix(new String[] { "flower", "flow", "flight" }));
        System.out.println("Nothing: " + solution.longestCommonPrefix(new String[] { "dog", "racecar", "car" }));
    }
}
